styles/index.css

```diff
@@ -0,0 +1,27 @@
+body {
+    font-family : Zilla slab;
```

Comment:
I'm not sure the font is being used, you probably need  to implement a @font-face. The rest looks good, goodjob!

Reply:
changed zilla_slab to koho